### PR TITLE
Fixing a typo

### PR DESCRIPTION
**Pull request by robertdosa**
## Description

Fixing a typo in the Related tutorials section

### CODE REVIEW

File: src/content/developers/docs/standards/tokens/index.md [hunk 1]
```diff
@@ -30,5 +30,5 @@ _Know of a community resource that helped you? Edit this page and add it!_
 
 - [Token integration checklist](/developers/tutorials/token-integration-checklist/) _– A checklist of things to consider when interacting with tokens._
 - [Understand the ERC20 token smart contract](/developers/tutorials/understand-the-erc-20-token-smart-contract/) _– An introduction to deploying your first smart contract on an Ethereum test network._
-- [Transfers and approval of ERC20 tokens from a Solidity smart contract](/developers/tutorials/transfers-and-approval-of-erc20-tokens-from-a-solidity-smart-contract/) _– AHow to use a smart contract to interact with a token using the Solidity language._
-- [Implementing an ERC721 market [a how-to guide]](/developers/tutorials/how-to-implement-an-erc721-market/) _– How to put tokenized items for sale on a decentralized classifieds board._
```
Comment (wackerow) [inline src/content/developers/docs/standards/tokens/index.md:34]: I believe this actually should be "classifieds" board... That's at least how I've always heard it, and distinguishes it from a "classified board" of directors.
Anyone else?

Comment (robertdosa) [reply src/content/developers/docs/standards/tokens/index.md:34]: You are right. "classifieds board" is the correct form. I'm not a native English speaker and it just felt weird, so I fixed it, but now that I looked it up I can confirm it is correct.

**Review by samajammin — APPROVED** [review]
Thanks for this!

**Comment by samajammin** [comment]
@all-contributors please add @robertdosa for content.

**Pull request merged**